src/main/kotlin/dev/binclub/paperbin/transformers/ElytraLowerRenderDistance.kt

```diff
@@ -0,0 +1,76 @@
+package dev.binclub.paperbin.transformers
+
+import dev.binclub.paperbin.PaperBinConfig
+import dev.binclub.paperbin.PaperFeature
+import dev.binclub.paperbin.utils.insnBuilder
+import dev.binclub.paperbin.utils.internalName
+import net.minecraft.server.v1_12_R1.*
+import org.objectweb.asm.Opcodes.*
+import org.objectweb.asm.tree.FieldInsnNode
+import org.objectweb.asm.tree.MethodInsnNode
+import org.objectweb.asm.tree.VarInsnNode
+
+/**
+ * @author Tigermouthbear 31/May/2020
+ */
+object ElytraLowerRenderDistance: PaperFeature {
+    @JvmStatic
+    private val distance = 2
```

Comment:
Perhaps instead of a hardcoded view distance of 2 we could maybe use the normal view distance - 2, or the normal view distance / 2? Not every server has a normal view distance of 4.

Reply:
Yea, in the discord I thought we said a default of 4 would be best